README.md

```diff
@@ -58,15 +58,21 @@ You can also run all the Pester tests by running `Invoke-Pester` in the `test` d
 
 Check out the [docs folder](https://github.com/PowerShell/Polaris/tree/master/docs) for the API reference.
 
+## Troubleshooting / Questions
+
+**Twitter:** #PSPolaris
+**Slack:** See us in the #Polaris channel of the [PowerShell Slack](http://slack.poshcode.org/)
+**Github Issues:** Submit a bug / new feature idea / or just a question as a new Github issue
+
 ## Roadmap
 
 We have a few paths we are interested in taking. We hope the community helps direct us.
 
-* Expanding on the current implementation using HttpListener to deliver features you'd expect from projects ASP.NET or Expressjs (route parameters, query parameters, middleware, auth etc)
+-   Expanding on the current implementation using HttpListener to deliver features you'd expect from projects ASP.NET or Expressjs (route parameters, query parameters, middleware, auth etc)
 
-* Investigating the use of [Kestrel](https://github.com/aspnet/KestrelHttpServer)/[ASP.NET Routing](https://github.com/aspnet/routing) instead of HttpListener
+-   Investigating the use of [Kestrel](https://github.com/aspnet/KestrelHttpServer)/[ASP.NET Routing](https://github.com/aspnet/routing) instead of HttpListener
```

Comment:
```suggestion
- Investigating the use of [Kestrel](https://github.com/aspnet/KestrelHttpServer)/[ASP.NET Routing](https://github.com/aspnet/routing) instead of HttpListener
```